src/doc/rustc/src/command-line-arguments.md

```diff
@@ -202,6 +202,10 @@ flag](codegen-options/index.md#extra-filename). The files are written to the
 current directory unless the [`--out-dir` flag](#option-out-dir) is used. Each
 emission type may also specify the output filename with the form `KIND=PATH`,
 which takes precedence over the `-o` flag.
+Specifying `-o -` or `--emit KIND=-` asks rustc to emit to stdout.
+This only works for text output types (`asm`, `dep-info`, `llvm-ir` and `mir`)
+unless stdout is not a tty. This will result in an error if multiple output types
```

Comment:
I think you mean to say: "This only works for text output types (`asm`, `dep-info`, `llvm-ir` and `mir`)
if stdout is not a tty."

Reply:
Yes, good catch. Even myself was confused. I reworded this.